### ISSUE TITLE
Confirm we are using latest Google Mobile Ads SDK for iOS (for Ad Manager)

### ISSUE THREAD

**Issue by waynedyck** [WSDOT/wsdot-ios-app#220]
Confirm we are using version 8.5.0 or later as those are built to drive key performance improvements.

Apple’s App Tracking Transparency (ATT) policy requires developers to ask for permission when they use certain information from other companies’ apps and websites for advertising purposes.

These changes will reduce visibility into key metrics that show how ads drive conversions (like app installs and sales) and will affect how advertisers value and bid on ad impressions. As such, app publishers may see a significant impact to their Google ad revenue on iOS with Apple’s ATT policies taking effect.

**Comment by waynedyck** [comment]
According to the `Podfile.lock` file it looks like we are using `Google-Mobile-Ads-SDK (7.69.0)`

**Comment by waynedyck** [comment]
Latest version of the [Google Mobile Ads SDK](https://developers.google.com/admob/ios/download) is 8.7.0

**Comment by waynedyck** [comment]
Log file confirms we are using, `afma-sdk-i-v7.69.0`

**Comment by waynedyck** [comment]
Our current Google-Mobile-Ads-SDK is installed as part of the `Firebase/AdMob` installation and, because of the logical operators, is currently limited to version 7.69.0
```
  - Firebase/AdMob (7.3.0):
    - Firebase/CoreOnly
    - Google-Mobile-Ads-SDK (~> 7.66)
```
Attempting to force an install of the latest version by adding, `pod 'Google-Mobile-Ads-SDK', '~> 8.7.0'` to the Podfile results in this error message:
```
[!] CocoaPods could not find compatible versions for pod "Google-Mobile-Ads-SDK":
  In Podfile:
    Firebase/AdMob was resolved to 7.3.0, which depends on
      Google-Mobile-Ads-SDK (~> 7.66)

    Google-Mobile-Ads-SDK (~> 8.7.0)
```
Need to look at what has changed with the Firebase installation and see about updating the Google-Mobile-Ads-SDK through it.

**Comment by waynedyck** [comment]
The [Prepare for SDK v8](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/migration) has information on changes as well as some renames and removals of APIs. The [Class renames](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/migration#class_renames) section is super helpful.
